### PR TITLE
Make address widget labels editable

### PR DESCRIPTION
**Pull request by dcsaszar**
https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-german.scrivito-portal-app.pages.dev/de?_scrivito_workspace_id=p5bcc7f596b13012&_scrivito_display_mode=diff

### CODE REVIEW

File: src/Widgets/AddressWidget/AddressWidgetComponent.tsx [hunk 1]
```diff
@@ -109,22 +106,28 @@ function Table({
...
+              <ContentTag
+                tag="span"
+                content={widget}
+                attribute={`${name}Label`}
+              />
+              :{' '}
+            </td>
             <td className="text-break">
-              <a href={`${LINK_PREFIXES[name]}:${value}`}>{value}</a>
+              <ContentTag
```
Comment (apepper) [inline src/Widgets/AddressWidget/AddressWidgetComponent.tsx:118]: Nice!

I only see, that in edit-mode this looks a bit broken design wise:

Screenshot 2024-04-11 at 14 26 12

**Review by apepper — COMMENTED** [review]
FEEDBACK ([201b806](https://github.com/Scrivito/scrivito-portal-app/pull/356/commits/201b806ef907bc9c70905354132e72b6dfa92c47))

Good stuff 🚀!

I only notice a small styling issue.

File: src/Widgets/AddressWidget/AddressWidgetComponent.tsx (hunk 1 above)
Comment (dcsaszar) [reply src/Widgets/AddressWidget/AddressWidgetComponent.tsx:118]: https://github.com/infopark/scrivito_js/issues/10820

**Review by apepper — APPROVED** [review]
LGTM ([201b806](https://github.com/Scrivito/scrivito-portal-app/pull/356/commits/201b806ef907bc9c70905354132e72b6dfa92c47))

**Pull request merged**